attributes/compose.rb

```diff
@@ -1,8 +1,8 @@
-default['osl-docker']['compose']['version'] = '1.18.0'
+default['osl-docker']['compose']['version'] = '2.15'
 default['osl-docker']['compose']['filename'] = "docker-compose-Linux-#{node['kernel']['machine']}"
 case node['kernel']['machine']
 when 'x86_64'
-  default['osl-docker']['compose']['checksum'] = 'b2f2c3834107f526b1d9cc8d8e0bdd132c6f1495b036a32cbc61b5288d2e2a01'
+  default['osl-docker']['compose']['checksum'] = 'ba481d45be2b137a2a185abd05f61d6d7766dbedfa038f16e4705760767a206e'
   default['osl-docker']['compose']['url_base'] = 'https://github.com/docker/compose/releases/download'
 when 'ppc64le'
   default['osl-docker']['compose']['checksum'] = '4458de249ca5f776738e44a6af2f869c1186b2d018dd15705045aa01a45e50c5'
```

Comment:
I just built a new version and pushed it up, please update the checksum to `2b57f69c438fadaa88dc383d8df8b98955a2ab9262c7ca2524102c8199c8efb4`